src/Stratis.Bitcoin.Features.Miner/BlockTemplate.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.Miner
+{
+    public sealed class BlockTemplate
+    {
+        public Block Block;
+
+        public List<Money> VTxFees;
+
+        public List<long> TxSigOpsCost;
+
+        public string CoinbaseCommitment;
+
+        public Money TotalFee;
+
+        public BlockTemplate()
+        {
+            this.Block = new Block();
```

Comment:
This is wrong mate (you will need to merge and use the factory to create a block)

Reply:
Yes, I have now merged and fixed 👍